testing/batch_processing/test_batch_processing.py

```diff
@@ -16,6 +16,7 @@
 
 # TODO: We can and should be verifying more results produced by this pipeline, but which values?
 TESTED_VALUES = [("t2/csa_c2c3.csv", 0, "MEAN(area)"),
+                 ("t2/csa_pmj.csv", 0, "MEAN(area)"),
```

Comment:
The current test checks the value against [cached result files](https://github.com/spinalcordtoolbox/spinalcordtoolbox/tree/master/testing/batch_processing/cached_results), so we would need to also commit the file `t2/csa_pmj.csv` generated by `batch_processing.sh` in order for the test to pass.

Reply:
oups,  forgot to push the commit! done in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3497/commits/5e77268f6d973c00ea1323928e56b92a16948305